metadata-ingestion/src/datahub/sdk/lineage_client.py

```diff
@@ -164,13 +580,14 @@ def add_dataset_copy_lineage(
 
         self._client.entities.update(updater)
 
+    # TODO: deprecate this method
```

Comment:
just use the @deprecated decorator - we have it in a couple other spots in the codebase

but also - maybe we can just remove this method?